libraries/scopes/src/simplify.js

```diff
@@ -0,0 +1,35 @@
+const {normalizeScopeSet, scopeCompare} = require('./normalize');
+
+exports.simplifyScopeExpression = (scope) => {
+  const values = (Object.values(scope)[0]);
+  if(duplicateExists(values)){
+    removeDuplicateScopes(scope);
+  }
```

Comment:
Normalizing will remove duplicates, so there's no need to do this too.

Reply:
Yes, exactly. So I can remove the whole function now. Thanks!